examples/non_fittable_mixin.py

```diff
@@ -0,0 +1,58 @@
+from typing import Any
+
+import numpy as np
+
+from neuraxle.base import BaseStep, NonTransformableMixin, NonFittableMixin
+from neuraxle.pipeline import Pipeline
+from neuraxle.steps.misc import BaseCallbackStep, TapeCallbackFunction
+
+
+def main():
+    # Fit method is automatically implemented as changing nothing.
+
+    class FitCallbackStep(NonTransformableMixin, BaseCallbackStep):
```

Comment:
could define classes above. 

Note: aren't those classes already defined inside Neuraxle? Is this duplicated code? Could you import this?

Reply:
I won't import it because this is an example